.github/ISSUE_TEMPLATE/usage_question.yaml

```diff
@@ -0,0 +1,66 @@
+name: Usage Question
+description: Ask a question
+labels: ["Type: usage"]
+assignees: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        While we enable issues as a mechanism for new contributors and passers-by who are unfamiliar with Apache Software Foundation projects to ask questions and interact with the project, we encourage users to ask such questions on public mailing lists:
+        
+        * Development discussions: dev@arrow.apache.org (first subscribe by sending an e-mail to dev-subscribe@arrow.apache.org).
+
+        * User discussions: user@arrow.apache.org (first subscribe by sending an e-mail to user-subscribe@arrow.apache.org).
+        
+        Do not be surprised by responses to issues raised here directing you to those mailing lists, or to report a bug or feature request here.
+
+        Thank you!
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the usage question you have. Please include as many useful details as possible.
+    validations:
+      required: true
+  - type: dropdown
```

Comment:
There is a [`multiple` option](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#dropdown) for this so which allows for multi-selection, I think it would make sense to enable this.